.github/workflows/cherryPick.yml

```diff
@@ -151,6 +151,14 @@ jobs:
             Engineering
             Hourly
 
+      - name: Assign the PR to the deployer
+        if: ${{ steps.cherryPick.outputs.SHOULD_AUTOMERGE == 'false' || steps.isPullRequestMergeable.outputs.IS_MERGEABLE == 'false' }}
+        uses: actions-ecosystem/action-add-assignees@a73fcabd82d847c5e7433fcfdd58ef9f7e8a3993
+        with:
+          number: ${{ steps.createPullRequest.outputs.pr_number }}
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          assignees: ${{ github.actor }}
```

Comment:
So this will only be correct for manual CP's (executed via the GH UI). When you CP by merging a PR with the `CP Staging` label, then `github.actor` will actually be `OSBotify`. In those cases, the "deployer" is the person who merged the PR, and that's who we need to assign here. In pseudocode:

```
if (github.actor === 'OSBotify') {
    // Need to add a new step to get more information about the PR we're CPing,
    // maybe replacing `getMergeCommitForPullRequest` with just a more generic `getPullRequest` action
    assignee = steps.getCPPullRequest.merged_by.login;
} else {
    assignee = github.actor;
}
```

Reply:
Oh, I see. I thought assigning the label triggered the action, then making the person who added it the `github.actor`.

Reply:
I just added two commits with the changes to do this, let me know what you think.